source/extension-manifest.json

```diff
@@ -207,14 +207,14 @@
                         "resultSelector": "jsonpath:$.[*]"
                     },
                     {
-                        "name": "OctopusAllProjects",
-                        "endpointUrl": "$(endpoint.url)/api/projects/all",
-                        "resultSelector": "jsonpath:$.[*]"
+                        "name": "OctopusListProjects",
```

Comment:
Moving away from our use of `/all` here, as the projects/all response is known for being chunky, and if you throw large descriptions into the mix, this maxes out the 2MB response size limit in Azure pretty quickly and breaks lots of things.